### PR TITLE
Fix ranks and numbers in Holman-2008-100

### PR DESCRIPTION
**Pull request by Kristina-Pianykh**
# Pull request checklist

- [ ] add new concept list
- [ ] add new metadata
- [ ] add new Concepticon concept sets
- [ ] add new Concepticon concept relations
- [ ] refine existing Concepticon concept set mappings
- [ ] refine Concepticon glosses
- [ ] refine Concepticon concept relations
- [ ] refine Concepticon concept definitions
- [ ] retire data

## Additional information
See the discussion in #843. Can I ask you @LinguList to have a quick look at it?

### CODE REVIEW

File: concepticondata/conceptlists.tsv [hunk 1]
```diff
@@ -1,5 +1,5 @@
...
+Zhao-2006-201	Zhào, Yànzhēn 趙燕珍	2006		201	basic	Chinese	Zhaozhuang Bai		Zhao2006		This is a concept list prepared to illustrate basic lexemes in Zhaozhuang Bai, a Southern Bai variety spoken in the South of Dali in Yunnan (China). The author intended to use Swadesh's [list of 200 items](:ref:Swadesh-1952-200), but there are a few derivations. Interestingly, some concepts are also identical (like the Chinese items for SKY and DAY (NOT NIGHT)), due to ambiguities in the glossing language. Furthermore, as in many Chinese lists, we observe a wrong translation from DULL, being rendered as STUPID rather than DULL of a knife.
```
Comment (LinguList) [inline concepticondata/conceptlists.tsv:2]: can you quickly check what changed here? Are there tabstops missing?

**Comment by Kristina-Pianykh** [comment]
Steinthal has already been merged earlier (see #840). The issues with conceptlists.tsv are fixed now.

**Comment by Kristina-Pianykh** [comment]
If there are no additional comments from your side @LinguList, I'd like to merge this PR to prevent it from dragging too much behind the master branch in the light of new changes.

**Comment by LinguList** [comment]
sorry, I was waiting for the other to be merged before, by @CarolinHu, but this has been done now. Thanks!

**Pull request merged**